ram/wram.asm

```diff
@@ -229,13 +236,31 @@ wca0a:: ds 1
 
 	ds 5 ; TODO
 
+wIntroJumptableIndex::
 wca10:: ds 1
+
+UNION
+wIntroBGMapPointer:: ds 2 ; ca11
```

Comment:
Please omit the address comments; the .sym file is where to check for those.

Reply:
Thanks for letting me know, will get rid of them